openpathsampling/sample.py

```diff
@@ -756,17 +760,24 @@ class Sample(StorableObject):
         The Ensemble this sample is drawn from
     """
 
+    parent = DelayedLoader()
+    # details = DelayedLoader()
+    mover = DelayedLoader()
+
     def __init__(self,
                  replica=None,
                  trajectory=None,
                  ensemble=None,
                  bias=1.0,
-                 details=None,
                  parent=None,
-                 mover=None
+                 mover=None,
+                 details=None
```

Comment:
> Correction. I already fixed this and we should be able to remove the `details=None` without problems.

Will it still load the old files (the ones in my Dropbox, used for testing)? Although maybe we should re-run them soon anyway, and call anything files older than 0.9.2 incompatible after some future point.